internal/pkg/cli/prompt/interactive/interactive_test.go

```diff
@@ -178,3 +178,94 @@ func TestPrompt_MultiSelectIndex(t *testing.T) {
 	assert.True(t, ok)
 	assert.Equal(t, []int{0, 2}, result)
 }
+
+func TestPrompt_ShowLeaveBlank(t *testing.T) {
+	t.Parallel()
+
+	// Create virtual console
+	stdout := testhelper.VerboseStdout()
+	console, _, err := testhelper.NewVirtualTerminal(t, expect.WithStdout(stdout), expect.WithCloser(stdout), expect.WithDefaultTimeout(5*time.Second))
+	assert.NoError(t, err)
+	p := interactive.New(console.Tty(), console.Tty(), console.Tty())
+
+	// Interaction
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		_, err := console.ExpectString("My input")
+		assert.NoError(t, err)
+
+		_, err = console.ExpectString("Leave blank for default value.")
+		assert.NoError(t, err)
+
+		time.Sleep(20 * time.Millisecond)
```

Comment:
Nešel by tady použít channel místo sleep? Všiml jsem si, že to tak dělají i ostatní testy, takže asi lepší to takhle nechat, ale ptám se jen pro zajímavost.

Reply:
Nejspíš jo, ale je to tu kvůli testu, tak tady je snaha to mít co nejjednodušší. (A navíc jsem jen kopíroval. 🙂)